Don't check for output on stderr to know if an import worked, host inputrc and
sysroot readline can cause warnings on stderr.

Upstream-Status: Backport (from autoconf-archive 883a2abd)
Signed-off-by: Ross Burton <ross.burton@intel.com>

diff --git a/m4/ax_python_devel.m4 b/m4/ax_python_devel.m4
index b990d5b..318b089 100644
--- a/m4/ax_python_devel.m4
+++ b/m4/ax_python_devel.m4
@@ -137,7 +137,7 @@ variable to configure. See ``configure --help'' for reference.
 	#
 	AC_MSG_CHECKING([for the distutils Python package])
 	ac_distutils_result=`$PYTHON -c "import distutils" 2>&1`
-	if test -z "$ac_distutils_result"; then
+	if test $? -eq 0; then
 		AC_MSG_RESULT([yes])
 	else
 		AC_MSG_RESULT([no])
